src/Arcus.Observability.Telemetry.Serilog.Enrichers/Extensions/LoggerEnrichmentConfigurationExtensions.cs

```diff
@@ -92,6 +92,36 @@ public static LoggerConfiguration WithVersion(this LoggerEnrichmentConfiguration
             return enrichmentConfiguration.With(new ApplicationEnricher(componentName, propertyName));
         }
 
+        /// <summary>
```

Comment:
Should we deprecate the other WithComponentName method ?
Or can we modify the implementation of the `WithComponentName` method that takes a string so that an `AppName` service is registered etc ?  That will be difficult I guess, since you do not have your service-collection there.

Reply:
Yes, that's right, it will be difficult as the logging is set up after the `AppName` is registered. Didn't deprecate this bc there are many valid configurations. You could also pass it here and don't use W3C, or pass in the name two times, or use a slightly different name for Microsoft technology. Also, if there's no need for automatic dependency tracking, you could also pass it here directly.